gui/build.py

```diff
@@ -80,14 +80,21 @@ def do_bundle(options):
         if os.path.isdir(dirname):
             shutil.rmtree(dirname)
     # create a intaller
+    if IS_WIN:
```

Comment:
Hm, I see. But when we call bundle under linux, the default argument will be also given to bundle if we forget to give a python path. So, why don't we set different default value of path when we call different bundle funtion?